lib/Dialect/FIRRTL/FIRRTLAnnotationHelper.cpp

```diff
@@ -349,7 +349,8 @@ InstanceOp firrtl::addPortsToModule(
 
 Value firrtl::borePortsOnPath(
     SmallVector<InstanceOp> &instancePath, FModuleOp lcaModule, Value fromVal,
-    StringRef newNameHint, InstancePathCache &instancePathcache,
+    StringRef newNameHint, Optional<Location> loc,
```

Comment:
Can this be `Optional<Location>`? I think `Location` is nullable / it may work with `Location{}` ?